### DISCUSSION TITLE
Is that a way to popup right-click menu?

### DISCUSSION THREAD

**Discussion by Fancyflame** [rust-windowing/winit#2734] [Q&A] [answered]
Like Edge does, and it is expected can be customized.
![image](https://user-images.githubusercontent.com/31250625/224681445-3c3b13d9-29f4-4117-821c-dee8167cd2e8.png)

**Comment by kchibisov** [comment, marked as answer]
Popup menus are not supported.

https://github.com/rust-windowing/winit/issues/403
